example/src/Scatter.purs

```diff
@@ -81,6 +81,15 @@ options sinData cosData = do
 
   E.series do
     E.scatter do
+      E.markPoint do

```

Comment:
I'm able to create a marked point by mimicking this code. I haven't been able to figure out how to use `markLine`, however.

Any guidance on say creating a horizontal line at the number 10.0, for example?